concepts/admin.py

```diff
@@ -8,17 +11,31 @@ def resolve(modeladmin, request, queryset):
         authorities.resolve(type(obj), obj)
 resolve.verbose_name = 'resolve selected concepts'
 
+def merge_concepts(self, request, queryset):
+    return render_to_response('admin/merge_concepts.html', {'concepts': queryset})
+    add_tag.short_description = "Merge Concepts"
```

Comment:
What is ``add_tag`` ? I think you want to set the ``verbose_name`` attribute of the function, like this (note the lack of indent, and lowercase):

```python
merge_concepts.verbose_name = 'merge concepts'
```